spec/models/work_activity_notification_spec.rb

```diff
@@ -17,18 +17,13 @@
     end
 
     it "does not enqueue an e-mail message to be delivered for the notification" do
+      user.disable_messages_from(collection: collection)
       described_class.create(user: user, work_activity: work_activity)
       expect(message_delivery).not_to have_received(:deliver_later)
     end
 
     context "when e-mail notifications are enabled for the Collection" do
```

Comment:
Connected to the comment just above: This context used to be the special case, but now it's the default case: Maybe move it out, and wrap the test just above with a context that disables messages?